### PR TITLE
fix!: removed IDA from vc datamodel

### PR DESCRIPTION
**Pull request by fmarino-ipzs**
## Content
This PR 

- temporarily removes the IDA profile from the data model sd-jwt vc. Information about authentic source verification data is ensured by PR #216  (in that PR the verification field has been added to the Issuer Metadata).
- introduces the status attestation

## Review

- [x] Ensure your files are written following RST specs (*not MD!*)
- [ ] Italian version
- [x] English version
- [x] Example files 
- [x] Ask for review

### CODE REVIEW

File: docs/en/pid-eaa-data-model.rst [hunk 1]
```diff
@@ -198,8 +138,7 @@ The ``claims`` parameter contains the User attributes with the following mandato
       - 
 
 
-
-PID Non-normative Examples
+PID non-normative examples
```
Comment (peppelinux) [inline docs/en/pid-eaa-data-model.rst]:
```suggestion
PID Non-Normative Examples
```

File: docs/en/pid-eaa-data-model.rst [hunk 2]
```diff
@@ -212,35 +151,16 @@ In the following, the non-normative example of a PID in JSON format.
     "jti": "urn:uuid:6c5c0a49-b589-431d-bae7-219122a9ec2c",
```
Comment (peppelinux) [inline docs/en/pid-eaa-data-model.rst]:
```suggestion
```

File: docs/en/pid-eaa-data-model.rst [hunk 3]
```diff
@@ -212,35 +151,16 @@ In the following, the non-normative example of a PID in JSON format.
...
-                "organization_id":
-                  "urn:eudi:it:organization_id:ipa_code:m_it",
-                "country_code": "IT"
-              }
-            }
-          }
-        ]
+    "status": {
+      "status_attestation": {
+        "credential_hash_alg": "S256"
```
Comment (peppelinux) [inline docs/en/pid-eaa-data-model.rst]:
```suggestion
        "credential_hash_alg": "sha-256"
```

File: docs/en/pid-eaa-data-model.rst [hunk 4]
```diff
@@ -262,32 +182,23 @@ The corresponding SD-JWT verson for PID is given by
...
+      "TK2RguPYoXzCx0vv5hbN9u5M2mHlWBt41qGWlLXCNu8",
+      "UHChpGtNF2bj1FvAfBby1rnf7WXkxelFJ5a4vSj2FO4",
+      "q6Tqnxau97tu-MqUDg0fSAmLGZdSuMUMk6a2s3bcsC0",
+      "wyfxVqq9BosPT7tN4SHOI4E48P19aVA1ktW5Zf0E-fc"
     ],
-    "iss": "https://issuer.example.org",
     "exp": 1883000000,
+    "iss": "https://pidprovider.example.org",
     "sub": "NzbLsXh8uDCcd7noWXFZAfHkxZsRGC9Xs",
     "jti": "urn:uuid:6c5c0a49-b589-431d-bae7-219122a9ec2c",
```
Comment (peppelinux) [inline docs/en/pid-eaa-data-model.rst]:
```suggestion
```

File: docs/en/pid-eaa-data-model.rst [hunk 5]
```diff
@@ -262,32 +182,23 @@ The corresponding SD-JWT verson for PID is given by
...
-        "_sd": [
-          "4g9lBt38U1EeTA1zlvvGfFgPPcoe3zmbQ_zSRDgHQaE",
-          "EYgzJ1hTYWJjhBK2V3b8HV3e_fEf-Udffc5ymY77WtQ",
-          "IcYHQydT_C3U1IqaJlFicxLlaHTHvElyFZ6Jxia27qQ",
-          "KxIG8rWXmtR884xTV7eXuHICfPYPw6gFvfr07v-d5oc",
-          "lXgxEDAuPeUvmkcNGr9FZuqodwFqUT01gJj7xd4yEPA"
-        ]
+    "status": {
+      "status_attestation": {
+        "credential_hash_alg": "S256"
```
Comment (peppelinux) [inline docs/en/pid-eaa-data-model.rst]:
```suggestion
        "credential_hash_alg": "sha-256"
```

File: docs/en/pid-eaa-data-model.rst [hunk 6]
```diff
@@ -432,37 +305,18 @@ In the following, we provide a non-normative example of (Q)EAA in JSON.
     "jti": "urn:uuid:6c5c0a49-b589-431d-bae7-219122a9ec2c",
```
Comment (peppelinux) [inline docs/en/pid-eaa-data-model.rst]:
```suggestion
```

File: docs/en/pid-eaa-data-model.rst [hunk 7]
```diff
@@ -432,37 +305,18 @@ In the following, we provide a non-normative example of (Q)EAA in JSON.
...
-        "given_name": "Mario",
-        "family_name": "Rossi",
-        "birth_date": "1980-01-10",
-        "expiry_date": "2024-01-01",
-        "tax_id_code": "TINIT-XXXXXXXXXXXXXXXX",
-        "constant_attendance_allowance": true
-      }
+    "status": {
+    "status_attestation": {
+      "credential_hash_alg": "S256"
```
Comment (peppelinux) [inline docs/en/pid-eaa-data-model.rst]:
```suggestion
      "credential_hash_alg": "sha-256"
```

File: docs/en/pid-eaa-data-model.rst [hunk 8]
```diff
@@ -484,34 +338,25 @@ The corresponding SD-JWT for the previous data is represented as follow, as deco
...
-          "EYgzJ1hTYWJjhBK2V3b8HV3e_fEf-Udffc5ymY77WtQ",
-          "F90SKK9nIQcHIElkHY_ult_9FGqYe-RydvY3E0qR96s",
-          "IcYHQydT_C3U1IqaJlFicxLlaHTHvElyFZ6Jxia27qQ",
-          "dfrmUvonZDgealZCGwk3ufmc_4ept3y9N7xhWZlCyxo",
-          "ji86HS1v3D41tU5JqW4oWCwTJDuTUwp1ewqoCUzzEXk",
-          "lXgxEDAuPeUvmkcNGr9FZuqodwFqUT01gJj7xd4yEPA"
-        ]
+    "status": {
+      "status_attestation": {
+        "credential_hash_alg": "S256"
```
Comment (peppelinux) [inline docs/en/pid-eaa-data-model.rst]:
```suggestion
        "credential_hash_alg": "sha-256"
```

**Comment by balanza** [comment]
It also introduces status attestations

File: docs/en/pid-eaa-data-model.rst [hunk 9]
```diff
@@ -429,40 +300,20 @@ In the following, we provide a non-normative example of (Q)EAA in JSON.
...
-        "family_name": "Rossi",
-        "birth_date": "1980-01-10",
-        "expiry_date": "2024-01-01",
-        "tax_id_code": "TINIT-XXXXXXXXXXXXXXXX",
-        "constant_attendance_allowance": true
-      }
+    "status": {
+    "status_attestation": {
+      "credential_hash_alg": "sha-256"
     }
```
Comment (peppelinux) [inline docs/en/pid-eaa-data-model.rst]:
```suggestion
    },
```

File: docs/en/pid-eaa-data-model.rst [hunk 10]
```diff
@@ -484,34 +335,25 @@ The corresponding SD-JWT for the previous data is represented as follow, as deco
...
+      "UHChpGtNF2bj1FvAfBby1rnf7WXkxelFJ5a4vSj2FO4",
+      "i9XHLePHyV8OM35l3nf1MKqfpWuD7OFpRamSAsX0-5g",
+      "rhPkItz7BGGpjnWX2SGVH_OV9VhRjz9Hx_INXwBbz6o",
+      "wyfxVqq9BosPT7tN4SHOI4E48P19aVA1ktW5Zf0E-fc"
     ],
-    "iss": "https://issuer.example.org",
     "exp": 1883000000,
+    "iss": "https://issuer.example.org",
     "sub": "NzbLsXh8uDCcd7noWXFZAfHkxZsRGC9Xs",
     "jti": "urn:uuid:6c5c0a49-b589-431d-bae7-219122a9ec2c",
```
Comment (peppelinux) [inline docs/en/pid-eaa-data-model.rst]:
```suggestion
```

File: docs/en/pid-eaa-data-model.rst [hunk 11]
```diff
@@ -15,16 +15,14 @@ The User attributes provided within the Italian PID are the ones listed below:
...
-
 The (Q)EAAs are issued by (Q)EAA Issuers to a Wallet Instance and MUST be provided in SD-JWT-VC or MDOC-CBOR data format. 
 
 The PID/(Q)EAA data format and the mechanism through which a digital credential is issued to the Wallet Instance and presented to a Relying Party are described in the following sections. 
 
 SD-JWT
 ======
 
-The PID/(Q)EAA is issued in the form of a digital credential. The digital credential format is `Selective Disclosure JWT format <https://datatracker.ietf.org/doc/html/draft-ietf-oauth-selective-disclosure-jwt-04>`_ as specified in `[draft-terbu-sd-jwt-vc-latest] <https://vcstuff.github.io/draft-terbu-sd-jwt-vc/draft-terbu-oauth-sd-jwt-vc.html>`__.
+The PID/(Q)EAA is issued in the form of a digital credential. The digital credential format is `Selective Disclosure JWT format <https://datatracker.ietf.org/doc/html/draft-ietf-oauth-selective-disclosure-jwt-04>`_ as specified in `[SD-JWT-based Verifiable Credentials 02] <https://www.ietf.org/archive/id/draft-ietf-oauth-sd-jwt-vc-02.html>`__.
```
Comment (peppelinux) [inline docs/en/pid-eaa-data-model.rst]:
```suggestion
The PID/(Q)EAA is issued in the form of a Digital Credential. The Digital Credential format is `Selective Disclosure JWT format <https://datatracker.ietf.org/doc/html/draft-ietf-oauth-selective-disclosure-jwt-04>`_ as specified in `[SD-JWT-based Verifiable Credentials 02] <https://www.ietf.org/archive/id/draft-ietf-oauth-sd-jwt-vc-02.html>`__.
```

File: docs/en/pid-eaa-data-model.rst [hunk 12]
```diff
@@ -44,7 +42,7 @@ The Disclosures are sent to the Holder together with the SD-JWT in the *Combined
 
   <Issuer-Signed-JWT>~<Disclosure 1>~<Disclosure 2>~...~<Disclosure N>
 
-See `[draft-terbu-sd-jwt-vc-latest] <https://vcstuff.github.io/draft-terbu-sd-jwt-vc/draft-terbu-oauth-sd-jwt-vc.html>`_ and `[SD-JWT] <https://datatracker.ietf.org/doc/html/draft-ietf-oauth-selective-disclosure-jwt-04>`__ for more details. 
+See `[SD-JWT VC] <https://www.ietf.org/archive/id/draft-ietf-oauth-sd-jwt-vc-02.html>`_ and `[SD-JWT] <https://datatracker.ietf.org/doc/html/draft-ietf-oauth-selective-disclosure-jwt-04>`__ for more details. 
 
 
 PID/(Q)EAA SD-JWT parameters
```
Comment (peppelinux) [inline docs/en/pid-eaa-data-model.rst]:
```suggestion
PID/(Q)EAA SD-JWT parameters:
```

**Review by peppelinux — APPROVED** [review]

**Review by grausof — APPROVED** [review]
LGTM

**Pull request merged**